### DISCUSSION TITLE
GitHub Actions broken again?

### DISCUSSION THREAD

**Discussion by almirsarajcic** [3Stones-io/omedis#308] [General]
The CI stopped running.

I don't think it's broken GitHub Actions this time. I suspect we might have run out of minutes. I don't have access so I can't tell for sure.

~~@wintermeyer, can you please check?~~ It works

<img width="884" alt="Screenshot 2024-11-09 at 11 36 56" src="https://github.com/user-attachments/assets/1b457a60-8b35-4120-bd1b-c118a6625489">
<img width="931" alt="Screenshot 2024-11-09 at 11 37 07" src="https://github.com/user-attachments/assets/e236160e-a122-4e6d-955f-98cd2861ccbf">
<img width="1071" alt="Screenshot 2024-11-09 at 11 37 23" src="https://github.com/user-attachments/assets/7050a8fc-2e9b-4458-9c6d-1d35b3f986dd">

**Comment by almirsarajcic** [comment]
Now it works. 🤷

I guess it was a case of PEBKAC - I just needed a little patience.

**Reply by wintermeyer** [reply]
We use the free version. Sometimes there is a hiccup. But not very often.

**Reply by almirsarajcic** [reply]
Yeah, that's understandable. Sorry for the mention. I should have just waited 😃
